contrib/go-redis/redis/redis.go

```diff
@@ -122,10 +122,11 @@ func (c *Pipeliner) execWithContext(ctx context.Context) ([]redis.Cmder, error)
 		tracer.ResourceName("redis"),
 		tracer.Tag(ext.TargetHost, p.host),
 		tracer.Tag(ext.TargetPort, p.port),
-		tracer.Tag("out.db", p.db),
+		tracer.Tag("out.db", strconv.Itoa(p.db)),
```

Comment:
Is it possible for the type change here to break an existing customer? (I noticed the other major versions of this already use a string here, it's curious this version doesn't)

Reply:
`out.db` should not change in this PR (I just changed when the string conversion is happening, since the new `db.redis.database_index` is an `int`).
Please let me know if this clarifies the question or if I missed something else.

Reply:
Oh I see, I missed that the strconv was happening before, just in a different spot. Thanks!